doc/whatsnew/2/2.16/index.rst

```diff
@@ -32,3 +32,654 @@ any experience with our code!
 Last but not least @clavedeluna and @nickdrozd became triagers, welcome to the team !
 
 .. towncrier release notes start
+
+What's new in Pylint 2.16.0?
+----------------------------
+Release date: 2023-01-15
```

Comment:
```suggestion
Release date: 2023-01-18
```
@DanielNoord what do you think ? I will be partially unavailable from January the 24th to the 30th so either we release around that time or we could release in February.

Reply:
18th sounds good to me!